### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by vickyhella**

### CODE REVIEW

**Review by niusmallnan — DISMISSED** [review]
LGTM

File: shell/assets/translations/zh-hans.yaml [hunk 1]
```diff
@@ -812,6 +812,7 @@ catalog:
     windowsIncompatible: 仅 Linux
     versionWindowsIncompatible: 仅限 Linux 版本
     header: Chart
+    featuredCharts: 功能 Chart
```
Comment (kingsd041) [inline shell/assets/translations/zh-hans.yaml:815]: `功能 Chart`--`精选 Chart`

**Comment by vickyhella** [comment]
Already updated per comments

**Review by kingsd041 — APPROVED** [review]
LGTM

**Review by niusmallnan — APPROVED** [review]

**Comment by niusmallnan** [comment]
ping  @richard-cox 
Could you help merge it into the 2.7.0 milestone?

**Review by catherineluse — APPROVED** [review]

**Pull request merged**